config/development_mysql.yaml

```diff
@@ -27,8 +25,6 @@ persistence:
         maxConns: 2
         maxIdleConns: 2
         maxConnLifetime: "1h"
-        encodingType: "thriftrw"
```

Comment:
Also remove those fields in postgres yaml and docker config template?

Reply:
yeah, removed.